rdflib_jsonld/context.py

```diff
@@ -9,21 +9,27 @@
 from rdflib.namespace import RDF
 
 from ._compat import basestring, unicode
-from .keys import (BASE, CONTAINER, CONTEXT, GRAPH, ID, INDEX, LANG, LIST,
-        REV, SET, TYPE, VALUE, VOCAB)
+from .keys import (BASE, CONTAINER, CONTEXT, GRAPH, ID, IMPORT, INCLUDED,
+        INDEX, JSON, LANG, LIST, NEST, NONE, PREFIX, PROPAGATE, PROTECTED, REV,
+        SET, TYPE, VALUE, VERSION, VOCAB)
 from . import errors
 from .util import source_to_json, urljoin, urlsplit, split_iri, norm_url
 
 
-NODE_KEYS = set([LANG, ID, TYPE, VALUE, LIST, SET, REV, GRAPH])
+NODE_KEYS = set([GRAPH, ID, INCLUDED, JSON, LIST, NEST, NONE, REV, SET, TYPE,
+                 VALUE, LANG])
 
 class Defined(int): pass
 UNDEF = Defined(0)
 
+# From <https://tools.ietf.org/html/rfc3986#section-2.2>
+URI_GEN_DELIMS = (':', '/', '?', '#', '[', ']', '@')
+
 
 class Context(object):
 
-    def __init__(self, source=None, base=None):
+    def __init__(self, source=None, base=None, version=None):
+        self.version = version or 1.0
```

Comment:
```suggestion
        self.version = version or 1.1
```
@niklasl - it seems that `"@version": 1.1` necessary to parse 1.1 docs. the [w3c proposed recommendation](https://www.w3.org/TR/json-ld/) (i know this happened in may 2020) says that `@version` should be used to force parsing as 1.0 rather than 1.1. therefore this suggestion is to be spec compliant.